trade/src/main/java/bisq/trade/bisq_easy/protocol/messages/BisqEasyTakeOfferRequestHandler.java

```diff
@@ -111,7 +112,7 @@ protected void verifyMessage(BisqEasyTakeOfferRequest message) {
                 .filter(offer -> offer.equals(takersOffer))
                 .findAny();
         if (matchingOfferInChannel.isEmpty()) {
-            BisqEasyTradeService bisqEasyTradeService = (BisqEasyTradeService) serviceProvider;
+            BisqEasyTradeService bisqEasyTradeService = ((TradeService) serviceProvider).getBisqEasyTradeService();
```

Comment:
The naming of TradeService and BisqEasyTradeService suggests a inheritance relationship, though thats not the case. TradeService is the root level service in the trade module, following the convention used in all domain modules. BisqEasyTradeService is the service for the BE protocol.  I don't have a good suggestion how to mitigate that confusing name relationship.

Instead of the cast we should add the `BisqEasyTradeService getBisqEasyTradeService();` getter to the ServiceProvider, thus not requiring the confusing cast.